coherence/upnp/devices/media_server.py

```diff
@@ -385,7 +385,12 @@ def process_child(self, ch, name, request):
                     self.warning(
                         f'{request.method} request with content-length '
                         f'{headers[b"content-length"]} header - sanitizing')
-                    del request.received_headers[b'content-length']
+					# This should fix #20
```

Comment:
You are [mixing spaces with tabs](https://travis-ci.org/opacam/Cohen3/jobs/571920850#L1463-L1466) (all the project uses tabs with 4 spaces per tab). Also the inline comment you put is missaligned with the try/except block....:thinking:... do you mind to be more explicit with the inline comment? so we can make life easier to any developer who touches those lines in a future :wink:. Something like:
```
                    # Fix headers for some devices (Samsung TVs, see issue #20)
                    try:
                        del request.received_headers[b'content-length']
                    ...
```